### ISSUE TITLE
Cannot import albumentations

### ISSUE THREAD

**Issue by starkfire** [albumentations-team/albumentations#1472]
## 🐛 Bug

After installing albumentations via Pip inside a project that uses .venv, the package can't be imported.

## To Reproduce

1. Run the following sample:

```py
import albumentations as A
```

Importing the package would yield the following:

```
  File "D:\Users\acer\Desktop\Projects\efficientnet-grain\datasets\aug.py", line 1, in <module>
    import albumentations as A
  File "D:\Users\acer\Desktop\Projects\efficientnet-grain\.venv\lib\site-packages\albumentations\__init__.py", line 5, in <module>
    from .augmentations import *
  File "D:\Users\acer\Desktop\Projects\efficientnet-grain\.venv\lib\site-packages\albumentations\augmentations\__init__.py", line 2, in <module>
    from .blur.functional import *
  File "D:\Users\acer\Desktop\Projects\efficientnet-grain\.venv\lib\site-packages\albumentations\augmentations\blur\__init__.py", line 1, in <module>
    from .functional import *
  File "D:\Users\acer\Desktop\Projects\efficientnet-grain\.venv\lib\site-packages\albumentations\augmentations\blur\functional.py", line 5, in <module>
    import cv2
  File "D:\Users\acer\Desktop\Projects\efficientnet-grain\.venv\lib\site-packages\cv2\__init__.py", line 181, in <module>
    bootstrap()
  File "D:\Users\acer\Desktop\Projects\efficientnet-grain\.venv\lib\site-packages\cv2\__init__.py", line 175, in bootstrap
    if __load_extra_py_code_for_module("cv2", submodule, DEBUG):
  File "D:\Users\acer\Desktop\Projects\efficientnet-grain\.venv\lib\site-packages\cv2\__init__.py", line 28, in __load_extra_py_code_for_module
    py_module = importlib.import_module(module_name)
  File "C:\Python39\lib\importlib\__init__.py", line 127, in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
  File "D:\Users\acer\Desktop\Projects\efficientnet-grain\.venv\lib\site-packages\cv2\typing\__init__.py", line 69, in <module>
    NumPyArrayGeneric = numpy.ndarray[typing.Any, numpy.dtype[numpy.generic]]
TypeError: 'numpy._DTypeMeta' object is not subscriptable
```

## Expected behavior

Should be able to import without any problem.

## Environment

 - Albumentations version: 1.3.1
 - Python version: 3.9.5
 - OS: Windows 10
 - How you installed albumentations: pip

## Additional context

The error also happens even if I force Albumentations to use the `opencv-python` and `opencv-python-headless` packages installed within the same project.

```sh
pip install -U albumentations --no-binary qudida,albumentations
```

**Comment by Baakchsu** [comment]
Hey @starkfire! I once ran into something similar because of dependency issues. Could you please try to manually remove all the pip dependency packages of albumentations that it installed when you ran `pip install albumentations`? 
After that, I would suggest you first do "pip install opencv-python==4.6.0.66". For some reason, this version installs quicker than the recent ones. Then, force albumentations using `pip install -U albumentations --no-binary qudida,albumentations`.

**Comment by starkfire** [comment]
@Baakchsu I already managed to fix the issue long ago, but unfortunately I can't reproduce it anymore, even in the case when I have simultaneous installations of `opencv-python` and `opencv-python-headless` (at v4.8.0.76). I'm thinking of closing this issue now as this issue is better explained at https://github.com/albumentations-team/albumentations/issues/1471. The issue has something to do with albumentations being incompatible with a previous version of opencv-python, but that issue no longer occurred to me when I used opencv-python at v4.8.0.76.
